cbt_import_whole_vdi.py

```diff
@@ -34,15 +48,25 @@ def main():
     parser.add_argument('-p', '--password', dest='password')
     parser.add_argument('-v', '--vdi-uuid', dest='vdi_uuid')
     parser.add_argument('-f', '--filename', dest='path')
+    parser.add_argument('--new', dest='new_vdi', action='store_const',

```

Comment:
Would this be more descriptive if this parameter was called --as-new-vdi?
Or can you please specify the help-parameter?

Reply:
ok to be safe I've done both